examples/finding_sunspots_using_stara.py

```diff
@@ -0,0 +1,94 @@
+"""
+============================
+Finding sunspots using STARA
+============================
+
+This example demonstrates the use of Sunspot Tracking And Recognition
+Algorithm (STARA) in detecting and plotting sunspots. More information
+on the algorithm can be found in [this](https://doi.org/10.1017/S1743921311014992) paper.
+If you wish to perform analysis over a large period of time we suggest to refer
+[this](https://gitlab.com/wtbarnes/aia-on-pleiades/-/blob/master/notebooks/tidy/finding_sunspots.ipynb)
+notebook implementation of the same algorithm using dask arrays.
+"""
+
+import astropy.units as u
+import matplotlib.pyplot as plt
+import sunpy.io._fits
+import sunpy.map
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+from sunkit_image.stara import stara
+
+###############################################################################
+# Firstly, let's download HMI continuum data from the Virtual Solar Observatory.
+
+query = Fido.search(a.Time("2023-01-01 00:00", "2023-01-01 00:01"), a.Instrument("HMI"), a.Physobs("intensity"))
```

Comment:
How would the output look if we used the HMI sample data from sunpy?

Reply:
The sample data present in sunpy is magnetogram, but the stara works better on continuum data. I've tried using magnetogram data to find the sunspots but it doesn't work.